.gitlab-ci.yml

```diff
@@ -53,6 +53,11 @@ stages:
 
 build:ts:
   image: $INTEGRATION_DOCKER_IMAGE
+  rules:
+    - if: $CI_COMMIT_BRANCH == 'master'
```

Comment:
@peterstarling it doesn't like this section.

Reply:
yeah try either
```
rules:
    - if: $CI_COMMIT_BRANCH == 'master'
    variables:
        REACT_APP_SENTRY_DSN: ${REACT_APP_SENTRY_DSN_DEV}
    - if: $CI_COMMIT_BRANCH != 'master'
 ```
 
 or just from inside the `script` definition remap the variable there